scripts/upload_incr_DB.py

```diff
@@ -27,17 +27,39 @@
 import socket
 import argparse
 from urllib import request, parse
+import logging
+from logging import handlers
 
 PERSISTENCE_SNAPSHOT_NAME='incremental'
 STATEDELTA_DIFF_NAME='statedelta'
 BUCKET_NAME='BUCKET_NAME'
 NUM_TXBLOCK = 1
 NUM_DSBLOCK= "PUT_INCRDB_DSNUMS_WITH_STATEDELTAS_HERE"
 NUM_FINAL_BLOCK_PER_POW= "PUT_NUM_FINAL_BLOCK_PER_POW_HERE"
-SOURCE = './'
 TESTNET_NAME= "TEST_NET_NAME"
 SYNC_INTERVAL = 1
 
+FORMATTER = logging.Formatter(
+    "[%(asctime)s %(levelname)-6s %(filename)s:%(lineno)s] %(message)s"
+)
+
+rootLogger = logging.getLogger()
+rootLogger.setLevel(logging.INFO)
+
+std_handler = logging.StreamHandler()
+std_handler.setFormatter(FORMATTER)
+rootLogger.addHandler(std_handler)
+
+def setup_logging():
+  logfile = os.path.dirname(os.path.abspath(__file__)) + "/upload_incr_DB-log.txt"
+  backup_count = 5
+  rotating_size = 8
+  fh = handlers.RotatingFileHandler(
+    logfile, maxBytes=rotating_size * 1024 * 1024, backupCount=backup_count
+    )
+  fh.setFormatter(FORMATTER)
+  rootLogger.addHandler(fh)
+
```

Comment:
will this impact or work as expected with the default compression that happens every `10 mins` ?